### DISCUSSION TITLE
Exception after installing Costum_nodes/Workflow-Component

### DISCUSSION THREAD

**Discussion by fifthinvent** [Comfy-Org/ComfyUI#3389] [Q&A]
Hello everybody, after installing the 'new_ComfyUI_windows_portable_nvidia_cu121_or_cpu' and costum_nodes/Workflow-Component it raised an exception in /ComfyUI_windows_portable/ComfyUI/custom_nodes/ComfyUI-Workflow-Component/workflow_component/component_loader.py.  >> Exception: ### [ERROR] "DF_Multiply" node is not installed. Please install that node through ComfyUI-Mananger. ### <<, please see below.
I have found the class "DF_Multiply" in sdxl-ksample.json file. KSampler(Advanced) - node is installed and seems to be ok.
But i can't find or install the DF_Multiply - node. 
(Is that maybe an issue not to be worth of mentioning, that will be solved within a further update?
   Btw, does DF means data-frame? What is the node's use?)

Best regards
"I'm very happy about such an amazing open source project, to bring AI into our homes. Thank you all for your work!"

# line 209, in get_input_types_dynamic
    build_input_types(i, input_mapping, input_types, node, node_config_map, spec_map)
       --> [Workflow-Component] '## sdxl-ksample [f14d94]' is broken. Maybe there are missing nodes. (INFO: ### [ERROR] "DF_Multiply" node is not installed. Please install that node through ComfyUI-Mananger. ###) <--
# line 357, in build_input_types
    raise Exception(errmsg)
     Exception: ### [ERROR] "DF_Multiply" node is not installed. Please install that node through ComfyUI-Mananger. ###

**Comment by theredwillow** [comment]
Here's what I did to make that error message go away... Installed **Derfuu ComfyUI ModdedNodes** from Manager.

I'm a noob to ComfyUI, so here's an explanation of why I did this.
First, I googled "df_multiply" comfyui.
I saw it mentioned here https://docs.getsalt.ai/md/ComfyUI-Advanced-ControlNet/
Tried installing that, didn't work. Turns out Google was just indexing the link to "DF Multiply" in the navigation bar.
Clicking into that link, you are taken to https://docs.getsalt.ai/md/Derfuu_ComfyUI_ModdedNodes/Nodes/DF_Multiply/?h=df_multiply#documentation

**Comment by Olietos** [comment]
Hello,
you need install nodes "Tiled sampling for ComfyUI", "Derfuu_ComfyUI_ModdedNodes","ComfyUI-Advanced-ControlNet"  through comfyui manager.
